chore: OPTIC-1337 Live Tooltips update for webinar

eurica opened the pull request:
Live tooltips update: Added details for the live event on December 10th, 2025

eurica committed:
chore: OPTIC-1337 Live Tooltips update for webinar

Added details for the live event on December 10th, 2025.

eurica committed:
Fix JSON formatting for liveContent.json

bmartel commented:
/fmt
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/19838031228)

bmartel committed:
Apply pre-commit linters

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/19838031228
